pegasus/helpers/form_helpers.rb

```diff
@@ -146,22 +146,27 @@ def insert_form(kind, data, options={})
   form_class = Object.const_get(kind)
   row[:source_id] = form_class.get_source_id(data) if form_class.respond_to? :get_source_id
 
+  # For the most recent HOC signups, a duplicate (matching email and name)
+  # entry is assumed to be an update rather than an insert.
+  num_rows_updated = 0
   if kind == 'HocSignup2016'
```

Comment:
Outside the scope of this PR, and not the highest priority, but why is form-specific functionality here in `insert_forms`? This breaks encapsulation and is a bad code smell. It should go inside the `HocSignup2016` form itself. It could go in the `normalize` method, which is called from `validate_form` on line 128, or if that's not appropriate we can add an interface for `on_insert` like we have for `process`. CC @tanyaparker